content/contributing/documentation/content_guidelines.rst

```diff
@@ -154,6 +155,28 @@ are also displayed in a dynamic *navigation bar* on the side.
    - :ref:`RST cheat sheet: headings <contributing/headings>`
    - :ref:`RST cheat sheet: specialized directives <contributing/specialized-directives>`
 
+.. _contributing/organizing-documentation:
+
+Organizing the documentation
+============================
+
+When writing documentation about a given topic, try to keep related pages organized.
+
+For most topics, a single page will do the job. Place it in the appropriate section of the
+documentation (e.g., content related to the CRM app go under :menuselection:`Applications
+-> Sales -> CRM`) and follow the :ref:`document structure <contributing/document-structure>`
+guidelines.
+
+| For a more complex topic, several pages—each discussing one of its aspects—may be needed.
```

Comment:
I would avoid the | to go to the line. This is more "French," while in English, different thoughts are separated into paragraphs.

I advise using it mostly to enhance the "layout" of your content.

```suggestion
For more complex topics, you may need several pages to cover all their aspects.
```